llvm/test/CodeGen/AMDGPU/barrier-elimination-gfx12.ll

```diff
@@ -0,0 +1,56 @@
+; RUN: llc -mtriple=amdgcn -mcpu=gfx1200 < %s | FileCheck %s
+; RUN: llc -mtriple=amdgcn -mcpu=gfx1200 < %s -global-isel | FileCheck %s
```

Comment:
move -global-isel to the front and use explicit =0 for the dag checks

Reply:
Done.